<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1" rowspan="4" colspan="5">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="6" column="6">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>переместить</string>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>удалить</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="10" column="5">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>повернуть</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>создать</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>изменить размер</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
